src/Compilers/CSharp/Test/Semantic/Semantics/SpanStackSafetyTests.cs

```diff
@@ -197,10 +197,10 @@ static void M3l(ref readonly SpanLike<string> ss)
     {
     }
 
-    // Not OK
```

Comment:
This was a bug. A small part of of previous safety rules that was not removed when rules were relaxed.
byref parameter is ref-returnable regardless of its type now.